### PR TITLE
[ja] Translate tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-deprecation-affects-you into Japanese

### LINKED ISSUE #31275: [ja] Translate tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-deprecation-affects-you into Japanese
# This is a Feature Request

## What would you like to be added

Translate [tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-deprecation-affects-you](https://kubernetes.io/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-deprecation-affects-you)  into Japanese.

## Why is this needed

This page has not been translated.

## Comments

/language ja
/assign

### PR DESCRIPTION
**Pull request by ptux**
resolve #31275

### CODE REVIEW

File: content/ja/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-deprecation-affects-you.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+content_type: task
+weight: 20
+---
+
+<!-- overview -->
+Kubernetesの`dockershim`コンポーネントは、DockerをKubernetesの{{< glossary_tooltip text="container runtime" term_id="container-runtime" >}}として使用することを可能にします。
+
+Kubernetesの組み込みコンポーネントである`dockershim`はリリースv1.20で非推奨となりました。
+
+このページでは、あなたのクラスタがどのようにDockerをコンテナランタイムとして使用しているか、使用中の`dockershim`が果たす役割について詳しく説明し、`dockershim`の廃止によって影響を受けるワークロードがあるかどうかをチェックするためのステップを示します。
```
Comment (nasa9084) [inline content/ja/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-deprecation-affects-you.md:12]:
```suggestion
このページでは、あなたのクラスターがどのようにDockerをコンテナランタイムとして使用しているか、使用中の`dockershim`が果たす役割について詳しく説明し、`dockershim`の廃止によって影響を受けるワークロードがあるかどうかをチェックするためのステップを示します。
```

please check below too

File: content/ja/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-deprecation-affects-you.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+## 自分のアプリがDockerに依存しているかどうかの確認 {#find-docker-dependencies}
+
+アプリケーション・コンテナの構築にDockerを使用している場合でも、これらのコンテナを任意のコンテナランタイム上で実行することができます。このようなDockerの使用は、コンテナランタイムとしてのDockerへの依存とはみなされません。
```
Comment (nasa9084) [inline content/ja/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-deprecation-affects-you.md:16]:
```suggestion
アプリケーションコンテナの構築にDockerを使用している場合でも、これらのコンテナを任意のコンテナランタイム上で実行することができます。このようなDockerの使用は、コンテナランタイムとしてのDockerへの依存とはみなされません。
```

File: content/ja/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-deprecation-affects-you.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+代替のコンテナランタイムが使用されている場合、Dockerコマンドを実行しても動作しないか、予期せぬ出力が得られる可能性があります。
+
+このように、Dockerへの依存があるかどうかを調べることができます:
+
+1. 特権を持つPodがDockerコマンド(`docker ps`など)を実行したり、Dockerサービスを再起動したり(`systemctl restart docker.service`などのコマンド)、Docker固有のファイル(`/etc/docker/daemon.json`など)を変更しないことを確認すること。
+1. Docker設定ファイル(`/etc/docker/daemon.json` など)にプライベートレジストリやイメージミラーの設定がないか確認します。これらは通常、別のコンテナランタイムのために再設定する必要があります。
```
Comment (nasa9084) [inline content/ja/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-deprecation-affects-you.md:23]:
```suggestion
1. Dockerの設定ファイル(`/etc/docker/daemon.json` など)にプライベートレジストリやイメージミラーの設定がないか確認します。これらは通常、別のコンテナランタイムのために再設定する必要があります。
```

File: content/ja/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-deprecation-affects-you.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+1. Kubernetesインフラストラクチャの外側のノードで実行されているスクリプトやアプリがDockerコマンドを実行しないことを確認します。それは、もしかしたら:
```
Comment (nasa9084) [inline content/ja/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-deprecation-affects-you.md:24]:
```suggestion
1. Kubernetesインフラストラクチャーの外側のノードで実行されているスクリプトやアプリがDockerコマンドを実行しないことを確認します。それは、もしかしたら:
```

File: content/ja/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-deprecation-affects-you.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+   - トラブルシューティングのためにノードにSSH接続します。
+   - ノードのスタートアップスクリプト。
+   - ノードに直接インストールされた監視エージェントやセキュリティエージェント。
+1. 上記のような特権的な操作を行うサードパーティツール。詳しくは[Migrating telemetry and security agents from dockershim](/docs/tasks/administer-cluster/migrating-from-dockershim/migrating-telemetry-and-security-agents) を参照してください。
+1. dockershimの動作に間接的な依存性がないことを確認します。
+   これはエッジケースであり、あなたのアプリケーションに影響を与える可能性は低いです。ツールによっては、Docker固有の動作に反応するように設定されている場合があります。例えば、特定のメトリクスでアラートを上げたり、トラブルシューティングの指示の一部として特定のログメッセージを検索したりします。そのようなツールを設定している場合、移行前にテストクラスタで動作をテストしてください。
+
+## Dockerへの依存について解説 {#role-of-dockershim}
+
+[コンテナランタイム](/docs/concepts/containers/#container-runtimes)とは、Kubernetes Podを構成するコンテナを実行できるソフトウェアです。
```
Comment (nasa9084) [inline content/ja/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-deprecation-affects-you.md:34]:
```suggestion
[コンテナランタイム](/ja/docs/concepts/containers/#container-runtimes)とは、Kubernetes Podを構成するコンテナを実行できるソフトウェアです。
```

File: content/ja/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-deprecation-affects-you.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+
+KubernetesはPodのオーケストレーションとスケジューリングを担当し、各ノードでは{{< glossary_tooltip text="kubelet" term_id="kubelet" >}}がコンテナランタイムインターフェイスを抽象化して使用するので、互換性があればどのコンテナランタイムでも使用することができます。
+初期のリリースでは、Kubernetesは1つのコンテナランタイムと互換性を提供していました:Dockerです。
```
Comment (nasa9084) [inline content/ja/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-deprecation-affects-you.md:37]:
```suggestion
初期のリリースでは、Kubernetesは1つのコンテナランタイムと互換性を提供していました: Dockerです。
```

File: content/ja/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-deprecation-affects-you.md [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+その後、Kubernetesプロジェクトの歴史の中で、クラスタ運用者は追加のコンテナランタイムを採用することを希望しました。
+CRIはこのような柔軟性を可能にするために設計され、kubeletはCRIのサポートを開始しました。
+しかし、DockerはCRI仕様が考案される前から存在していたため、Kubernetesプロジェクトはアダプタコンポーネント「dockershim」を作成しました
```
Comment (nasa9084) [inline content/ja/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-deprecation-affects-you.md:40]:
```suggestion
しかし、DockerはCRI仕様が考案される前から存在していたため、Kubernetesプロジェクトはアダプタコンポーネント「dockershim」を作成しました。
```

**Review by nasa9084 — CHANGES_REQUESTED** [review]

**Comment by ptux** [comment]
@inductor

cloud you give an LGTM label?

**Review by inductor — CHANGES_REQUESTED** [review]

File: content/ja/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-deprecation-affects-you.md [hunk 8]
```diff
@@ -0,0 +1,62 @@
...
+代替のコンテナランタイムが使用されている場合、Dockerコマンドを実行しても動作しないか、予期せぬ出力が得られる可能性があります。
+
+このように、Dockerへの依存があるかどうかを調べることができます:
+
+1. 特権を持つPodがDockerコマンド(`docker ps`など)を実行したり、Dockerサービスを再起動したり(`systemctl restart docker.service`などのコマンド)、Docker固有のファイル(`/etc/docker/daemon.json`など)を変更しないことを確認すること。
+1. Dockerの設定ファイル(`/etc/docker/daemon.json` など)にプライベートレジストリやイメージミラーの設定がないか確認します。これらは通常、別のコンテナランタイムのために再設定する必要があります。
+1. Kubernetesインフラストラクチャーの外側のノードで実行されているスクリプトやアプリがDockerコマンドを実行しないことを確認します。それは、もしかしたら:
+   - トラブルシューティングのためにノードにSSH接続します。
+   - ノードのスタートアップスクリプト。
+   - ノードに直接インストールされた監視エージェントやセキュリティエージェント。
```
Comment (inductor) [inline content/ja/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-deprecation-affects-you.md:27]:
```suggestion
1. Kubernetesインフラストラクチャーの外側のノードで実行される以下のようなスクリプトやアプリがDockerコマンドを実行しないことを確認します。
   - トラブルシューティングのために人間がノードにSSH接続
   - ノードのスタートアップスクリプト
   - ノードに直接インストールされた監視エージェントやセキュリティエージェント
```

**Review by inductor — CHANGES_REQUESTED** [review]

**Pull request merged**